Where_Is_My_Wife/Assets/_Main/Scripts/StateMachine/StateManager.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Zenject;
+
+public abstract class StateManager<EState> : MonoBehaviour where EState : Enum
```

Comment:
Rename `PlayerStateManager`

Reply:
There's a class called PlayerStateMachine, this is the template class